insonmnia/hub/config.go

```diff
@@ -0,0 +1,57 @@
+package hub
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+type HubConfig struct {
+	GRPCEndpoint  string   `yaml:"grpc_endpoint"`
+	MinerEndpoint string   `yaml:"miner_endpoint"`
+	Bootnodes     []string `yaml:"bootnodes"`
```

Comment:
If Bootnodes has special format, it might be better to define a custom type with Unmarshaller